sec1_intro_and_setup/1.1_basics.rb

```diff
@@ -0,0 +1,17 @@
+#Section 1.1
+#Let's code the quiz!
+
+#Creating an array
+address = [1,2,3,4,5,6,7,8,9]
+
+#printing array
+p address
+
+#printing array with index 5
+p address[5]
+
+#assigning new_address to store the reverse of the array
+new_address = address.reverse!
```

Comment:
???

Reply:
I didn't update the branch with main. Now it's updated.